README.md

```diff
@@ -77,6 +77,34 @@ Available since Fedora 32.
 `dnf install puddletag`
 </details>
 
+<details>
+<summary><b>Nix</b></summary>
+
+Available for channels 24.05 and unstable
+
+On NixOS:
+```
+nix-env -iA nixos.puddletag
+```
+
+On Non NixOS:
+```
+# without flakes:
+nix-env -iA nixpkgs.puddletag
+# with flakes:
+nix profile install nixpkgs#puddletag
```

Comment:
i dont use nix: is this really a pound sign `#` or is that supposed to be a dot? the rest seems to use a dot `.`

Reply:
Yes, that command is different because when we are using profiles to install with flakes, the program "name" is marked with a `#`. Yeah, little bit strange, I agree 😅, but that's the syntax we use when dealing with profiles in Nix.
https://nix.dev/manual/nix/2.22/command-ref/new-cli/nix3-profile-install
![image](https://github.com/user-attachments/assets/98c1167d-a818-4633-aedd-d65fd1872067)